CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
 
-## v7.0.0 31th May 2024
+## v7.0.0 12th Jun 2024
```

Comment:
Is this correct?

Reply:
Yeah, that's just an estimated release date